src/models/generated/GENROU.jl

```diff
@@ -0,0 +1,185 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+"""
+    mutable struct GENROU <: Machine
```

Comment:
The names GenSal, GenRou we should include them in the docstrings as the name in PSS/e but we need to give them a proper name (SalientPoleExponentialSaturation or something like that) and add a reference to Sauer Pai.

Reply:
I'm not sure exactly if I agree in this one. GENROU, GENSAL and those naming are in a way the standard approach of naming those. I understand that here we are only referring to the machine itself and not the generator, but going with this naming could be confusing. Let's discuss more on this.